server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -493,37 +512,38 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
         const dataContent = this.extractDataContent(message);
 
         // Check if we should nack this message
-        const nackMessages = this.nackMessages;
-        if (nackMessages && this.serviceConfiguration.deli.enableNackMessages) {
-            let shouldNack = true;
-
-            if (nackMessages.allowSystemMessages &&
-                (isServiceMessageType(message.operation.type) || !message.clientId)) {
-                // this is a system message. don't nack it
-                shouldNack = false;
-            } else if (nackMessages.allowedScopes) {
-                const clientId = message.clientId;
-                if (clientId) {
-                    const client = this.clientSeqManager.get(clientId);
-                    if (client) {
-                        for (const scope of nackMessages.allowedScopes) {
-                            if (client.scopes.includes(scope)) {
-                                // this client has an allowed scope. don't nack it
-                                shouldNack = false;
-                                break;
+        if (this.nackMessages.size > 0 && this.serviceConfiguration.deli.enableNackMessages) {
+            for (const nackMessages of this.nackMessages.values()) {
```

Comment:
Should we rename this to `nackMessageControlMessageContents`? It is a long name, and I know the original code also used `nackMessages`, but reading the code for the first time made me confused between `nackMessages` and `this.nackMessages`. I think renaming might make it easier to understand it.

Reply:
Sounds good. I renamed it